worlds/sc2/item_names.py

```diff
@@ -365,118 +369,128 @@
+ABERRATION_BANELING_INCUBATION                    = "Baneling Incubation (Aberration)"
+ABERRATION_PROTECTIVE_COVER                       = "Protective Cover (Aberration)"
+ABERRATION_RESOURCE_EFFICIENCY                    = "Resource Efficiency (Aberration)"
+CORRUPTOR_MONSTROUS_RESILIENCE                    = "Monstrous Resilience (Corruptor)"
+CORRUPTOR_CONSTRUCT_REGENERATION                  = "Construct Regeneration (Corruptor)"
+CORRUPTOR_SCOURGE_INCUBATION                      = "Scourge Incubation (Corruptor)"
+CORRUPTOR_RESOURCE_EFFICIENCY                     = "Resource Efficiency (Corruptor)"
+PRIMAL_IGNITER_CONCENTRATED_FIRE                  = "Concentrated Fire (Primal Igniter)"
+PRIMAL_IGNITER_PRIMAL_TENACITY                    = "Primal Tenacity (Primal Igniter)"
+OVERLORD_IMPROVED_OVERLORDS                       = "Improved Overlords (Overlord)"
+OVERLORD_VENTRAL_SACS                             = "Ventral Sacs (Overlord)"
+OVERLORD_GENERATE_CREEP                           = "Generate Creep (Overlord)"
+OVERLORD_PNEUMATIZED_CARAPACE                     = "Pneumatized Carapace (Overlord)"
+OVERLORD_ANTENNAE                                 = "Antennae (Overlord)"
+INFESTED_SCV_BUILD_CHARGES                        = "Sustained Cultivation Ventricles (InfSCV Build Charges)"
+INFESTED_MARINE_PLAGUED_MUNITIONS                 = "Plagued Munitions (Infested Marine)"
+INFESTED_MARINE_RETINAL_AUGMENTATION              = "Retinal Augmentation (Infested Marine)"
+INFESTED_BUNKER_CALCIFIED_ARMOR                   = "Calcified Armor (Infested Bunker)"
+INFESTED_BUNKER_REGENERATIVE_PLATING              = "Regenerative Plating (Infested Bunker)"
+INFESTED_BUNKER_ENGORGED_BUNKERS                  = "Engorged Bunkers (Infested Bunker)"
+TYRANNOZOR_BARRAGE_OF_SPIKES                      = "Barrage of Spikes (Tyrannozor)"
+TYRANNOZOR_TYRANTS_PROTECTION                     = "Tyrant's Protection (Tyrannozor)"
+TYRANNOZOR_HEALING_ADAPTATION                     = "Healing Adaptation (Ultralisk/Tyrannozor)"
+TYRANNOZOR_IMPALING_STRIKE                        = "Impaling Strike (Ultralisk/Tyrannozor)"
+NYDUS_WORM_OMEGA_WORM_SUBTERRANEAN_SCALES         = "Subterranean Scales (Nydus Worm/Omega Worm)"
+NYDUS_WORM_OMEGA_WORM_JORMUNGANDR_STRAIN          = "Jormungandr Strain (Nydus Worm/Omega Worm)"
+NYDUS_WORM_OMEGA_WORM_RESOURCE_EFFICIENCY         = "Resource Efficiency (Nydus Worm/Omega Worm)"
+NYDUS_WORM_RAVENOUS_APPETITE                      = "Ravenous Appetite (Nydus Worm)"
+OMEGA_WORM_OUROBOROS_STRAIN                       = "Ouroboros Strain (Omega Worm)"
+INFESTED_SIEGE_TANK_PROGRESSIVE_AUTOMATED_MITOSIS = "Progressive Automated Mitosis (Infested Siege Tank)"
+INFESTED_SIEGE_TANK_ACIDIC_ENZYMES                = "Acidic Enzymes (Infested Siege Tank)"
+INFESTED_SIEGE_TANK_DEEP_TUNNEL                   = "Deep Tunnel (Infested Siege Tank)"
+FRIGHTFUL_FLESHWELDER                             = "Frightful Fleshwelder (Zerg)"
```

Comment:
Put this item into Evolution Pit group (Zerg misc upgrades)

Reply:
done